src/Provider/DataSourceProvider.js

```diff
@@ -26,6 +26,7 @@ export function parseSourceData(data, extDest, layer) {
         withNormal: layer.isGeometryLayer !== undefined,
         withAltitude: layer.isGeometryLayer !== undefined,
         layers: source.layers,
+        styles: source.styles,
         style: layer.style,
```

Comment:
I am not a fan of having`styles` and `style` on the same object. Any idea on an alternative here ?

Reply:
yes, it will move in future PR.